tests/dualtor/test_standby_tor_upstream_mux_toggle.py

```diff
@@ -37,11 +37,14 @@ def test_standby_tor_upstream_mux_toggle(
     toggle_all_simulator_ports, set_crm_polling_interval):              # noqa F811
     itfs, ip = rand_selected_interface
     PKT_NUM = 100
+
+    logging.debug("PAUSE_TIME: %s PKT_NUM: %s" % (PAUSE_TIME, PKT_NUM))
```

Comment:
is this debug required?

Reply:
Not necessary but It makes easy for everyone to understand/debug while executing the testcase.